jmclient/jmclient/blockchaininterface.py

```diff
@@ -413,7 +413,7 @@ def estimate_fee_per_kb(self, N, quiet=False):
         for estimation of feerate by Core, or a number of satoshis
         per kilo-vbyte (see `fee_per_kb_has_been_manually_set` for
         how this is distinguished).
-        In both cases it is prevented for falling below the current
+        In both cases it is prevented form falling below the current
```

Comment:
:) I think you meant "from"

Reply:
Ahhh... twice typos in four characters! :D